src/NuGet.Core/NuGet.Packaging/NuspecReader.cs

```diff
@@ -554,6 +554,14 @@ public bool GetRequireLicenseAcceptance()
             return StringComparer.OrdinalIgnoreCase.Equals(bool.TrueString, GetMetadataValue("requireLicenseAcceptance"));
         }
 
+        /// <summary>
+        /// Read package dependencies for all frameworks
+        /// </summary>
+        public IEnumerable<FrameworkSpecificGroup> GetFrameworkRefGroups()
```

Comment:
I would appreciate any feedback on the name here. 

`GetFrameworkReferenceGroups` already exists for the "FrameworkAssemblies".

Reply:
I really don't like this name but I have no better one, as the good was already taken by FrameworkAssemblies.

Maybe `GetFrameworkReferenceGroupsV2`?

Reply:
How about `GetSdkFrameworkReferenceGroups`?

---
In reply to: [267143418](https://github.com/NuGet/NuGet.Client/pull/2745#discussion_r267143418) [](ancestors = 267143418)

Reply:
I've usually avoided NetCore/SDK mentions in the naming. But what you are proposing is probably better than what I currently have.